src/overlays/actors/ovl_En_Bigpo/z_en_bigpo.c

```diff
@@ -188,7 +188,7 @@ void EnBigpo_Init(Actor* thisx, PlayState* play2) {
 
     Actor_ProcessInitChain(&this->actor, sInitChain);
 
-    this->switchFlags = BIG_POE_GET_SWITCHFLAGS(thisx);
+    this->switchFlags = BIG_POE_GET_SWITCH_FLAG(thisx);
```

Comment:
This actor still stores the value as `switchFlags` (plural)

Reply:
It seems to be still storing only 1 flag? So I would change `this->switchFlags` to `this->switchFlag`. Or do you prefer `switchFlags` in general?

Reply:
No, I had a bad habit in the past of calling it `switchFlags` even if it was only holding one flag, I mean it should be changed to match the macro now, `switchFlag`.